src/ncp/ncp_base.cpp

```diff
@@ -1091,6 +1092,7 @@ void NcpBase::LinkRawReceiveDone(otRadioFrame *aFrame, otError aError)
     SuccessOrExit(error = OutboundFrameSend());
 
 exit:
+    OT_UNUSED_VARIABLE(error);
```

Comment:
I kept these variables because I had in mind, that someone might want to use them for debugging purposes. But I can remove them and update the PR if we agree to do so.